src/ex4/server/services/item_manager.js

```diff
@@ -0,0 +1,89 @@
+const PokemonClient = require("../clients/pokemon_client");
+const { Item } = require("../db/models/item");
+
+class ItemManager {
+  constructor() {
+    this.pokemonClient = new PokemonClient();
+  }
+
+  getItems = () => {
+    try {
+      return Item.findAll();
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  handleItem = async (item) => {
+    console.log("item manager - handleItem", item);
+    if (this._isNumber(item)) {
+      return await this.fetchAndAddPokemon(item);
+    }
+    if (this._isList(item)) {
+      return await this.fetchAndAddManyPokemon(item);
+    }
+
+    this.addItem(item);
+  };
+
+  addItem = (item) => {
+    try {
+      console.log("item manager - add item:", { item });
+      Item.create({
+        id: id,
+        ItemName: item,
+        createdAt: new Date(),
```

Comment:
no need to manually modify createdAt or updatedAt:)

reminder:
<img width="1370" alt="image" src="https://user-images.githubusercontent.com/54766936/176713744-37c6cedb-9424-4389-8b80-34cf29494b9f.png">